index.html

```diff
@@ -41,20 +44,155 @@ <h1 id="nm">I’m Mariane<br><br>
                 <div class="socials">
                     <ul>
                         <img src="./assets/Vector.svg" alt="">
-                        <li><a href=""></a></li>
+                        <li><a href="https://twitter.com/oussedit"></a></li>
                         <img src="./assets/Vector (1).svg" alt="">
                         <li><a href=""></a></li>
                         <img src="./assets/Vector (2).svg" alt="">
-                        <li><a href=""></a></li>
+                        <li><a href="https://github.com/codedit334"></a></li>
                         <img src="./assets/Vector (3).svg" alt="">
                         <li><a href=""></a></li>
                     </ul>
```

Comment:
[OPTIONAL]  To make this section much more closer to the Figma template, I recommend adjusting the spacing between the social Icons and the let's connect text

![hmmm](https://user-images.githubusercontent.com/84930355/212093382-239a5612-9012-479b-a161-4f069f50721d.PNG)